### DISCUSSION TITLE
Ploting problem

### DISCUSSION THREAD

**Discussion by Adelija** [maximtrp/tmplot#4] [General]
Hello Maxim,
I'm trying to use the tmplot package from python but I can't run the visualization. Do you know if maybe the problem is the version or something else? All requirements and dependencies are fine. I am able to get coords and print top_docs
```
topics_coords = tmp.prepare_coords(model)
print(topics_coords.head())

print(tmp.get_top_docs(docs=texts, model=model))

```
but the line
`tmp.report(model=model, docs=texts, height=400, width=250)
`
doesn't give any plots.
Thanks in advance

**Comment by maximtrp** [comment]
Hello Adelija,
Are you using `tmplot` with Jupyter Lab?

**Reply by Adelija** [reply]
No, I am using it in python script in the spyder environment.

**Reply by maximtrp** [reply]
Sorry, but you cannot use the report interface in Spyder. This is available only in Jupyter.

**Reply by Adelija** [reply]
Oh, thanks. I will use Jupyter instead. Thank you, Maxim, that helps a lot. All the best, Adela

**Reply by maximtrp** [reply]
You are welcome! :)
